### PR TITLE
Add GraphQL automock tooling

### PR DESCRIPTION
**Pull request by OzzieOrca**
- Add graphql-ergonomock
- Add helper function to make graphql-ergonomock support fragments and generated types
- Add ContactRow test
- Add contacts page test

### CODE REVIEW

File: src/components/Contacts/ContactRow.stories.tsx [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import { Story, Meta } from '@storybook/react';
 import { withDesign } from 'storybook-addon-designs';
+
+import { gqlMock } from '../../../__tests__/util/graphqlMocking';
 import { ContactRow } from './ContactRow';
-import { ContactRowFragment } from './ContactRow.generated';
+import {
+  ContactRowFragment,
+  ContactRowFragmentDoc,
```
Comment (TheNoodleMoose) [inline src/components/Contacts/ContactRow.stories.tsx:9]: So does this file get generated by graphqlMocking file?

Comment (OzzieOrca) [reply src/components/Contacts/ContactRow.stories.tsx:9]: No, this is from codegen. We write the `.graphql` file and it generates an export with the contents of the `.graphql` file wrapped in `gql` from `graphql-tag`.
`src/components/Contacts/ContactRow.generated.ts` contains:
```ts
export const ContactRowFragmentDoc = gql`
  fragment ContactRow on Contact {
    ...
  }
`;
```

**Review by TheNoodleMoose — APPROVED** [review]
This looks good to me. Should we try to find a fix for the issue you mentioned in this pr or address it in a future pr and just get this merged in?

**Comment by OzzieOrca** [comment]
I'm just going to fix it later (hopefully sometime this week or next).

**Pull request merged**